website/docs/procs.md

```diff
@@ -108,7 +108,7 @@ to connect the `yield` call to the `blk` parameter automatically.
 
 Many Ruby constructs accept a block argument in one context, but then execute it
 in a different context entirely. This means that inside the block, certain
-methods will definitely exist that definitely don't exist immediately above the
+methods may exist that don't exist immediately above the
```

Comment:
I thought the `definitely` was confusing. A block may or may not contain methods that are not defined in the scope it is called in.